src/code42cli/output_formats.py

```diff
@@ -37,46 +35,20 @@ def extraction_output_format(_, __, value):
     return to_table
 
 
-format_option = click.option(
-    "-f",
-    "--format",
-    type=click.Choice(OutputFormat(), case_sensitive=False),
-    help="The output format of the result. Defaults to table format.",
-    callback=output_format,
-)
-
-
-extraction_format_option = click.option(
-    "-f",
-    "--format",
-    type=click.Choice(OutputFormat(), case_sensitive=False),
-    help="The output format of the result. Defaults to table format.",
-    callback=extraction_output_format,
-)
-
-
-def to_dynamic_csv(output, header):
```

Comment:
@kiran-chaudhary No need for dynamic_csv, as csv always is dynamic now (no custom headers, always gets all the data).  To get a subset of the data, you have to use TABLE format